app/models/google_calendar/google_event.rb

```diff
@@ -0,0 +1,18 @@
+module GoogleCalendar
+  class GoogleEvent
+    extend Listing
+    extend Adding
+    extend Deleting
+
```

Comment:
Hmmm... I was thinking more about adding some features to our GoogleEvent model by extending some utility modules so we can now say something like: "GoogleEvent can list, delete, add events". However, if this is not a good way I will refactor it :+1: